libs/shared/ui/src/lib/components/accordions/accordion-organization/accordion-organization.tsx

```diff
@@ -0,0 +1,69 @@
+import { useProjects } from '@console/domains/projects'
+import { Accordion, Icon } from '@console/shared/ui'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export interface AccordionOrganizationProps {
+  open?: boolean
+  name: string
+  logo?: string | null
+  className?: string
+  organizationId: string
+}
+
+export function AccordionOrganization(props: AccordionOrganizationProps) {
+  const { open = false, name, logo, className = '', organizationId } = props
+
+  const { projects, getProjects } = useProjects()
```

Comment:
You cannot call the domain project here, can you comment on this component because we don't use it, and it generate tests errors